.ci/docker/common/install_pip_requirements.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -ex
+
+python3.10 -m venv /opt/venv/tutorials
+source /opt/venv/tutorials/bin/activate
+pip install --upgrade pip
+pip install -r ./requirements.txt
+chown ci-user:ci-user /opt/venv -R
```

Comment:
Why do we need a venv here? It's our docker container, why not install dependencies system-wide?

Reply:
Removed